lntest/harness_node.go

```diff
@@ -768,6 +684,28 @@ func (hn *HarnessNode) start(lndBinary string, lndError chan<- error,
 		return err
 	}
 
+	return nil
```

Comment:
If we don't keep track of the PID file, do we even need to write it in the first place?